src/cache-restore.ts

```diff
@@ -1,19 +1,24 @@
-// Copyright 2023 The MathWorks, Inc.
+// Copyright 2023-2024 The MathWorks, Inc.
 
 import * as cache from '@actions/cache';
 import * as core from '@actions/core';
 import * as crypto from "crypto";
 import { State } from './cache-state';
 import { Release } from './matlab';
 
-export async function restoreMATLAB(release: Release, platform: string, architecture: string, products: string[], matlabPath: string): Promise<boolean> {
+export async function restoreMATLAB(release: Release, platform: string, architecture: string, products: string[], matlabPath: string, supportPackagesPath?: string): Promise<boolean> {
     const installHash = crypto.createHash('sha256').update(products.sort().join('|')).digest('hex')
     const keyPrefix = `matlab-cache-${platform}-${architecture}-${release.version}`;
     const primaryKey = `${keyPrefix}-${installHash}`;
-    const cacheKey: string | undefined = await cache.restoreCache([matlabPath], primaryKey);
+    const cachePaths = [matlabPath]
```

Comment:
Semicolon, also for line 10.

Does MathWorks have any specific style guidelines for typescript?